features/features.go

```diff
@@ -41,6 +42,7 @@ var features = map[FeatureFlag]bool{
 	RandomDirectoryEntry:     false,
 	IPv6First:                false,
 	DirectoryMeta:            false,
+	RegistrationsPerIPRange:  false,
```

Comment:
The flag-gating here is actually not needed because rate limit policies have their own enablement flag. Specifically, if there is no section in the config for a given policy, rlPolicy.Enabled() will return false. So we can deploy this without a change to rate-limit-policies.yml.

It's worth noting a subtle thing: While this meets the deployability requirements (can be deployed without concurrent config changes), it actually introduces a change in behavior immediately on deploy: instead of querying for, and rate limiting on, the /48, it will immediately start doing an exact query, and rate limiting on that. I think you are correct here in making the judgement call that this small change in behavior is safe enough to take effect immediately upon deploy.